code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -913,6 +913,20 @@
 	M.adjustToxLoss(current_cycle*rand(2,4))
 	..()
 
+/datum/reagent/phalloidin
+	name = "Phalloidin"
+	id = "phalloidin"
+	description = "A toxin produced by Destroying Angel mushrooms. Deadly, stealthy, and impossible to purge."
+	reagent_state = LIQUID
+	color = "#E7E7E7"
+	taste_mult = 0
+	unable_to_purge = TRUE
+	invisible_to_scanners = TRUE
+
+/datum/reagent/phalloidin/on_mob_delete(mob/living/M)
+	M.adjustToxLoss(current_cycle*rand(2,4))
```

Comment:
Balance wise, perhaps tune this down to `rand(1,3)` and allow it to do minor brain damage.